command/info.go

```diff
@@ -78,15 +98,35 @@ func newInfoEvent(t domain.Issue) flyte.Event {
 	}
 }
 
-type infoSuccessPayload struct {
-	Id          string `json:"id"`
-	Summary     string `json:"summary"`
-	Status      string `json:"status"`
-	Description string `json:"description"`
-	Assignee    string `json:"assignee"`
+func parseInput(input json.RawMessage) (string, error) {
```

Comment:
This is an overly generic method name. It would be clearer if it were something like _parseJiraIDFromInput_ or something specific so I know exactly what to expect this method to produce from the input.